inc/types/class-base.php

```diff
@@ -93,7 +106,7 @@ public function handle_authorisation() {
 
 		$submit = wp_unslash( $_POST['wp-submit'] );
 
-		$data = compact( 'redirect_uri', 'scope', 'state' );
+		$data = compact( 'redirect_uri', 'scope', 'state', 'code_challenge', 'code_challenge_method' );
```

Comment:
These should have default values (or just merge `$pkce_data` if it's set)

Reply:
The rest of the code assumes if there is no ```code_challenge``` set, PKCE is not being used. If PKCE is forced, it will throw an error before (just added that). I'll merge ```$pkce_data``` if set.